### PR TITLE
Remove json fallback

### PR DESCRIPTION
**Pull request by dr3**
~~Required for #1276~~ EDIT, not sure how that issue is related.

**Overall change:** Removes json fallback for unknown blocks

**Code changes:**

- Removes json fallback for unknown blocks
- Updates snapshots

This can be tested by looking at fixtures, such as 9 which had json output previously

---

- [x] I have assigned myself to this PR and the corresponding issues
- [x]  Tests added for new features
- [x] Test engineer approval
- [x] I have followed [the merging checklist](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/MERGE_PROCESS.md) and this is ready to merge.

### CODE REVIEW

File: src/app/containers/Blocks/index.jsx
Comment (sareh) [inline src/app/containers/Blocks/index.jsx]: This is clearer if you return null early & check for a `componentsToRender` and `type` both exist first.

e.g.
```
if (!componentsToRender || !type || !Block) { 
  return null; 
}
```

Also, I know that there is a snapshot with part of it capturing the null case, but it would be much better to have an explicit test case that checks `Blocks` returns exactly `null` when given a random block type.

**Review by sareh — CHANGES_REQUESTED** [review]
It's great to get rid of the JSON objects. I've added notes about testing and refactoring.

File: src/app/containers/Blocks/index.test.jsx
Comment (sareh) [inline src/app/containers/Blocks/index.test.jsx]: It would be clearer to rename this, since this isn't an `unknown` nested block - all of the blocks here are known. It is just that they are not within the `componentsToRender` for this case. 

```suggestion
const parentBlockNotAllowed = {
```

File: src/app/containers/Blocks/index.test.jsx
Comment (sareh) [inline src/app/containers/Blocks/index.test.jsx]: Clarification of what the output should be would be good, especially given how our we update our snapshots quite regularly
```suggestion
      'should render known blocks; unknown blocks rendered null',
```

File: src/app/containers/Blocks/index.test.jsx
Comment (dr3) [reply src/app/containers/Blocks/index.test.jsx]: done :)

**Review by sareh — APPROVED** [review]
👍 Super! Looks beautiful locally. No more JSON output. 🗑

**Review by twinlensreflex — APPROVED** [review]
Looks good locally for me 👏 🎉 😁

**Comment by ChrisBAshton** [comment]
If I go into the fixture data and change one of the `paragraph` blocks to `foo`, I get warnings in the console:

Screen Shot 2019-03-14 at 09 05 00

I suspect we want to fail silently rather than litter the console. What do you think?

Otherwise works well 👍 would appreciate your documented thoughts before I hit the merge button!

**Comment by dr3** [comment]
@ChrisBAshton Thats happening on latest, need to investigate when that started, this should be good to merge

**Comment by ChrisBAshton** [comment]
Yes it’s happening on latest but for different reasons. We can fix up latest but it's trickier to know we’ve fixed this up as we don’t have any fixture data for it, other than hacking it locally. This one specifically happens when you change from a known to an unknown block type.

Can you raise a separate issue to fix this one later? I'm then happy to merge 👍

**Comment by dr3** [comment]
Done :) https://github.com/bbc/simorgh/issues/1416

**Pull request merged**
